spec/path-scanner-spec.coffee

```diff
@@ -126,7 +126,7 @@ describe "PathScanner", ->
           expect(paths).not.toContain path.join(rootPath, 'sample.js')
           expect(paths).toContain path.join(rootPath, 'sample.txt')
 
-      it "local directory inclusions override global directory exclusions", ->
+      it "allows a local directory inclusion to override a matching global exclusion", ->
```

Comment:
If we could also test an inclusion that's *inside* an exclusion, I'd feel more comfortable merging this. So if `foo/bar` is excluded but we search for `foo/bar/baz` it should show up... it's not clear to me that will work from the current code.